rollup.config.js

```diff
@@ -31,10 +31,10 @@ const replaceValues = {
         "process.env.AP_ROOM_BASE_URL": JSON.stringify(process.env.AP_ROOM_BASE_URL),
         "process.env.RTCSTATS_URL": JSON.stringify(process.env.RTCSTATS_URL || "wss://rtcstats.srv.whereby.com"),
         "process.env.REACT_APP_API_BASE_URL": JSON.stringify(
-            process.env.REACT_APP_API_BASE_URL || "https://api.whereby.dev"
+            process.env.REACT_APP_API_BASE_URL || "https://embedded-ip-127-0-0-1.hereby.dev:4090"
```

Comment:
why don't you just use the env var instead?

Reply:
I think I did this after attempting to use the env var for a few seconds and failing

Pure laziness 😅